### PR TITLE
The Next Web home page clone  merge request

### PR DESCRIPTION
**Pull request by 8Bts**
We created a branch called "homepage" and collaborated on that branch.

- [x] On the initial commit, we created a basic structure for the page using semantics.

- [x] "Mobile first" approach is used in order to achieve responsiveness principles.

- [x] Page's content sections are made one by one for _mobile_, _tablet_, and _desktop_ versions separately. 

- [x] Elements are positioned using Grid, FlexBox, and Float

Please review our project before we merge our branch.

Thanks in advance!

### CODE REVIEW

**Comment by sumancrest0001** [comment]
**Changes required**
**Extraordinary work @temesghentekeste :clap :clap**

Your layout looks very much similar to that of the original and it is responsive at defined breakpoints.
I couldn't have asked more than this but there is two minor issues that need to be addressed to make it perfect. I know they are very minor and I am sure that it won't take much time.

[Required] It seems that you forget to add title on the third article of the main section.
Currently :point_down: :point_down: 
Screenshot from 2020-04-25 13-40-55

It should have a  title (like APP INSIGHT) before the heading :point_down: 
Screenshot from 2020-04-25 13-40-28

**Comment by sumancrest0001** [comment]
[Required] Let's remove some bottom margins from the articles, especially first and last. It will help us to align the last line of the article at the bottom as in the original site.
Currently :point_down: 
Screenshot from 2020-04-25 13-51-31

It should be like this :point_down: 
Screenshot from 2020-04-25 13-51-03

**Comment by sumancrest0001** [comment]
[Required] We have to fix the images sizes of the latest news section on a small screen. Currently, image size is different for each article which disturbed the alignment of titles also. 
on your project :point_down: 
Screenshot from 2020-04-25 13-51-48

On the original site :point_down: 
Screenshot from 2020-04-25 13-52-04

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,933 @@
...
+  justify-content: center;
+}
+
+.flex-center-h {
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  justify-content: center;
+}
+
```
Comment (sumancrest0001) [inline css/style.css]: [Optional] Even though other CSS stylings are perfect. Can we just remove the repetition of styling between these two classes? Let's follow the DRY principle. [pathright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) (reference) [reference](https://speckyboy.com/good-bad-css-practicesl)

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
**Kindly make the above-requested changes and request for another review.**

**Happy coding !!**
Have any questions? Contact me on slack **Suman Shrestha**

**Comment by temesghentekeste** [comment]
Thanks, @sumancrest0001 we will follow your review and make the necessary amendments.

**Comment by lekegitrepo** [comment]
#### Hi @8Bts and @temesghentekeste :clap: :+1: :100: 
#### Good job :muscle:  for implementing the required changes mentioned in the previous review.
#### This project is approved :1st_place_medal: 

### Reviewed by Temitayo Adeleke.

- Github: [@lekegitrepo](https://github.com/lekegitrepo)
- Twitter: [@leketemi](https://twitter.com/leketemi)
- Linkedin: [linkedin](https://www.linkedin.com/in/temitayo-adeleke/)

**Review by lekegitrepo — APPROVED** [review]
#### Status: Approved.

**Comment by temesghentekeste** [comment]
Thanks, @lekegitrepo for approving the project.

**Pull request merged**